Final tutorial level

clayton-snyder opened the pull request:
Fixes #102 (WIP)

* Jump, strafe, and killzone introduced first; have to jump to avoid the lava and get to the next platform

* Shooting grapple and rappelling up/down introduced next; have to shoot the point and rappel up to get to the next platform.

* Swinging/releasing introduced next; have to swing over the next wall to move on

* Bouncy platforms introduced next. Can bounce over the next wall or use the grapple point to get across.

* Lower half of level allows the player to bounce around and play with chain-swinging on grapple hooks. Bouncy floor makes it easier and faster to move around so players can more conveniently practice the swing mechanic before they move on.


**Estimated:** 45 **Min:** 30 **Max:** 60 **Actual:** Not finished yet

clayton-snyder commented:
Need to make sure that there is no wall jump possibility.

The first wall that's too high to jump over has this issue.

clayton-snyder commented:
add a button on the main menu to access the level

Chris-Johnston commented:
Awesome. I think that this level is the right level of difficulty for the tutorial (and that's none at all).

I would remove the "you win" sprite that appears when the player wins, since we are immediately loading the next scene as soon as they win.

I was able to notice it for a few frames before the next scene loaded.

Chris-Johnston commented on 385/ProjectSettings/EditorBuildSettings.asset:
```diff
@@ -41,4 +41,7 @@ EditorBuildSettings:
     path: Assets/Scenes/TutorialFinal.unity
     guid: 111a8aacc7a8148c7911ee4fb6676c42
+  - enabled: 1
+    path: Assets/Scenes/RealFinalTutorial.unity
```
That naming scheme tho.

Chris-Johnston commented:
I would move the climing instruction towards the end when climbing is needed, or at least space it out from the point it's next to. I would also add some label that explains what the red and the blue are. Otherwise I think that the images are a good improvement.

clayton-snyder commented:
Idea was to have a wall too high at the beginning so they had to climb up to get there, but there does seem to be too many instructions crowded at the beginning.

So I moved the climb instruction to the end and changed the rest of the level so they shouldn't need to know how to climb to get past the parts before the end.

Also added labels to win/lose zones.

SaamIAm commented:
Adjust the camera so when 'c' is pressed I can see the entire level and not have portions of it cut off.

Chris-Johnston commented:
LGTM :shipit:
